app/models/master_file.rb

```diff
@@ -548,7 +548,7 @@ def create_working_file!(full_path)
   end
 
   def self.calculate_working_file_path(old_path)
-    config_path = Settings.matterhorn.media_path
+    config_path = Rails.application.secrets.matterhorn_client_media_path
```

Comment:
UAL update these paths from secrets.yml

Reply:
Also some usecase I don't know, separate into a client and server path to allow client and server to have different nfs mount points for the same item....